UPGRADE-7.0.md

```diff
@@ -83,6 +110,25 @@ project requires WYSIWYG JavaScript, allow it by setting the configuration key
 
 The RevisionManagementBundle was removed. Remove it from the AppKernel.
 
+# Still-deprecated Code Entities
+
+Some code entities have been marked as deprecated in previous releases, but were not removed in this release because of
+wide usage, bug risks or quite a short deprecation period. They will be removed in a future Chameleon release, so be
+sure to remove calls over time.
+
+Entities that were NOT removed include (incomplete list):
+
+- Any database fields
+- Almost any method arguments
+- Classic main menu
+- Anything logging-related
```

Comment:
-> "All logging-related classes and service definitions"

Reply:
Also properties and methods - isn't it more or less OK as it is?